app/views/organisations/_corporate_information.html.erb

```diff
@@ -1,11 +1,22 @@
-<div class="organisation-corporate-info">
-  <%= render "govuk_publishing_components/components/heading", {
-      text: "Corporate Information",
-      heading_level: 4
-  } %>
-  <% corporate_info.each do |info| %>
-    <p>
-      <%= link_to info["title"], info["href"] %>
-    </p>
+<div class="column-one-third">
+  <% if @show.corporate_information[:corporate_information_links][:items].any? %>
```

Comment:
Should this IF statement encompass the wrapping column-one-third div?

Reply:
I don't think so - the check in `show_organisation` checks if there's any corporate information before rendering this partial, so it won't render if there's nothing. In here, we just want to check whether we're rendering corporate information links or job links, but at this point we know there's at least one.